modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/BasicInfo/BasicInfo.tsx

```diff
@@ -104,14 +80,6 @@ export function BasicInfo({
 	>([]);
 
 	const [creationLanguageId, setCreationLanguageId] = useState<Locale>();
```

Comment:
Hey @alinedoleron could you please rename this state to `creationLanguageId2` and `setCreationLanguageId`?